src/test_utils.rs

```diff
@@ -31,16 +37,18 @@ pub(crate) fn issuer_params(org_name: impl Into<String>) -> rcgen::CertificatePa
         rcgen::KeyUsagePurpose::DigitalSignature,
         rcgen::KeyUsagePurpose::CrlSign,
     ];
-    ca_params.alg = RCGEN_SIGNATURE_ALG;
     ca_params
 }
 
 pub(crate) fn end_entity_params(subject_alt_names: Vec<String>) -> rcgen::CertificateParams {
-    let mut ee_params = rcgen::CertificateParams::new(subject_alt_names);
+    let mut ee_params = rcgen::CertificateParams::new(subject_alt_names).unwrap();
     ee_params.is_ca = rcgen::IsCa::ExplicitNoCa;
-    ee_params.alg = RCGEN_SIGNATURE_ALG;
     ee_params
 }
 
+pub(crate) fn make_keypair() -> rcgen::KeyPair {
```

Comment:
I would probably skip this, can just use `rcgen::KeyPair::generate()` (which you are already using in some places, so better to be uniform)?

Reply:
SGTM. Removed and updated all the `generate_for` callsites to use the same algo.